docs/source/user-guide/python.md

```diff
@@ -28,9 +28,25 @@ popular file formats files, run it in a distributed environment, and obtain the
 
 The following code demonstrates how to create a Ballista context and connect to a scheduler.
 
+If you are running a standalone cluster (runs locally), all you need to do is call the stand alone cluster method `standalone()` or your BallistaContext. If you are running a cluster in remote mode, you need to provide the URL `Ballista.remote("http://my-remote-ip:50050")`.
+
 ```text
->>> import ballista
->>> ctx = ballista.BallistaContext("localhost", 50050)
+>>> from ballista import Ballista, BallistaBuilder
+>>> # for a standalone instance
+>>> # Ballista will initiate with an empty config
+>>> # set config variables with `set()`
+>>> ballista = BallistaBuilder()\
+>>>    .set("ballista.job.name", "example ballista")\
+>>>    .set("ballista.shuffle.partitions", "16")\
+>>>    .set("ballista.executor.cpus", "4")\
+>>>    .build()
+>>> 
+>>> # Show the Ballista Config
+>>> print(ballista.show_config())
```

Comment:
i would recommend remove `show_config` method, it is not needed

Reply:
removed show_config